specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+      "description": "A paged collection of classification policies.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PagedClassificationPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CloseJobRequest": {
+      "description": "Request payload for completing jobs",
+      "required": [
+        "assignmentId"
+      ],
+      "type": "object",
+      "properties": {
+        "assignmentId": {
+          "description": "The assignment within which the job is to be closed.",
+          "maxLength": 50,
+          "minLength": 0,
+          "type": "string"
+        },
+        "dispositionCode": {
+          "description": "Indicates the outcome of the job, populate this field with your own custom values.",
+          "maxLength": 500,
+          "minLength": 0,
+          "type": "string"
+        },
+        "releaseTime": {
+          "format": "date-time",
+          "description": "If not provided, capacity will be released immediately.\r\nIf provided, the future time at which to release the capacity.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "note": {
+          "description": "(Optional) A note that will be appended to the jobs' Notes collection with th current timestamp.",
+          "maxLength": 2000,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "CloseJobResponse": {
+      "type": "object"
+    },
+    "CompleteJobRequest": {
+      "description": "Request payload for completing jobs",
+      "required": [
+        "assignmentId"
+      ],
+      "type": "object",
+      "properties": {
+        "assignmentId": {
+          "description": "The assignment within the job to complete.",
+          "maxLength": 50,
+          "minLength": 0,
+          "type": "string"
+        },
+        "note": {
+          "description": "(Optional) A note that will be appended to the jobs' Notes collection with th current timestamp.",
+          "maxLength": 2000,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "CompleteJobResponse": {
+      "type": "object"
+    },
+    "ConditionalQueueSelector": {
+      "description": "Describes a set of label selectors that will be attached if the given condition resolves to true",
+      "required": [
+        "condition",
+        "labelSelectors"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/QueueSelectorAttachment"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "readOnly": true
+        },
+        "condition": {
+          "description": "The condition that must be true for the label selectors to be attached"
```

Comment:
Please specify a `type` for this property.  All properties should specify a `type`.  Please add `type` to any property where it is currently missing.

Reply:
Tracked using [issue](https://skype.visualstudio.com/SPOOL/_workitems/edit/2788073)